package.json

```diff
@@ -56,15 +60,23 @@
       "default": "./lib/browser/interceptors/WebSocket/index.js"
     },
     "./RemoteHttpInterceptor": {
-      "browser": null,
       "types": "./lib/node/RemoteHttpInterceptor.d.ts",
+      "node": {
```

Comment:
Should we have "default" here too? I wonder when does it fall through "default" 🤔 If it's nether `require` not `import`? So, in the current state, it will find no "node.default", and fallback to the "/node.default" on the export path level?

Reply:
I think any tool that will apply an export condition like "node" will also apply "import" or "require". I would probably not pollute too much now until someone comes around the corner with an example where it's missing.